src/server.c

```diff
@@ -1335,18 +1379,24 @@ int serverCron(struct aeEventLoop *eventLoop, long long id, void *clientData) {
                 used = kvstoreSize(server.db[j].keys);
                 vkeys = kvstoreSize(server.db[j].expires);
                 if (used || vkeys) {
-                    serverLog(LL_VERBOSE, "DB %d: %lld keys (%lld volatile) in %lld slots HT.", j, used, vkeys, size);
+                    serverLog(LL_VERBOSE, "DB %d: Total %lld keys (%lld/%lld volatile keys) in %lld slots HT.", j, used,
+                              vkeys, used, size);
                 }
             }
         }
     }
 
     /* Show information about connected clients */
     if (!server.sentinel_mode) {
+        char hmem[64];
+        size_t zmalloc_used = zmalloc_used_memory();
+
+        bytesToHuman(hmem, sizeof(hmem), zmalloc_used);
+
         run_with_period(5000) {
-            serverLog(LL_DEBUG, "%lu clients connected (%lu replicas), %zu bytes in use",
+            serverLog(LL_DEBUG, "Total: %lu clients connected (%lu replicas), %s(%zu bytes) in use",
```

Comment:
I would prefer to keep this a little closer to the current implementation.
```suggestion
            serverLog(LL_DEBUG, "Total: %lu clients connected (%lu replicas), %zu (%s) bytes in use",
```

Just so that it's a bit more similar for someone that expects it in a given format.

Reply:
I think it adds a little more value to the format given.